Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>Consider making access_threshold a slider widget for ease to changing.</p><p> </p><p>q as variable name again.</p><p> </p><p>Best to check aspx_data is a populated DataFrame after the query</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Really like the slider idea, I've implemented it. Q's are fixed, checks added to the dataframe :)</p>